### PR TITLE
Fix issues with IHNC (newton-mod) Integral equation method

### PR DESCRIPTION
**Pull request by marvinbernhardt**
I messed up some signs when rewriting that part to catch some rare events. This is the fix.

### CODE REVIEW

**Comment by marvinbernhardt** [comment]
@votca-bot help

**Comment by marvinbernhardt** [comment]
> @votca-bot help

Ah I thought this might work^^. How do I make the bot to add something to the changelog?

**Comment by junghans** [comment]
@votca-bot changelog: fix issues with IHNC (newton-mod) Integral equation method

**Comment by junghans** [comment]
@felipeZ please review!

File: share/scripts/inverse/iie.py [hunk 1]
```diff
@@ -442,24 +442,18 @@ def calc_dU_newton(r, g_tgt, g_cur, G_minus_g, n, kBT, rho, cut_off,
...
-            with np.errstate(divide='ignore', invalid='ignore', under='ignore'):
-                jac_inv1 = kBT * (1 - np.log(g_tgt / g_cur) / Delta_g)
-            jac_inv2 = kBT * dcdg
+                jac_inv1 = kBT * (1 + np.log(g_tgt / g_cur) / Delta_g)
+            jac_inv2 = -kBT * dcdg
             # Some fixes, because we want to define a jacobian matrix
             # Unmodified Newton is less awkward
-            # Ensure this is zero on the diagonal where Delta_g is zero
+            # Ensure this is zero, not nan, on the diagonal where Delta_g is zero
             jac_inv1[Delta_g == 0] = 0
```
Comment (felipeZ) [inline share/scripts/inverse/iie.py:455]: Is `delta_g` in this case an integer? Comparison to zero with floating points may generate unexpected issues.

**Review by felipeZ — APPROVED** [review]
LGTM

File: share/scripts/inverse/iie.py (hunk 1 above)
Comment (marvinbernhardt) [reply share/scripts/inverse/iie.py:455]: Ah no, it is float. You're right this is not ideal.

**Pull request merged**